frontend/src/metabase/common/components/EntityPicker/components/ResultItem/ResultItem.unit.spec.tsx

```diff
@@ -216,4 +221,17 @@ describe("EntityPicker > ResultItem", () => {
       screen.getByText(`in ${tableItem.database_name}`),
     ).toBeInTheDocument();
   });
+
+  it("should display table schema when available (metabase#44460)", () => {
```

Comment:
could we also add a test when there is an empty schema

Reply:
52b63798a5fb095b2b942ebc6d2bc90b9ca7c7f7